src/Features/CSharp/Portable/CodeRefactorings/EnableNullable/EnableNullableCodeRefactoringProvider.cs

```diff
@@ -252,6 +272,33 @@ private enum CodeActionPurpose
             Apply,
         }
 
+        private sealed class CustomFixAllProvider : FixAllProvider
+        {
+            public static readonly CustomFixAllProvider Instance = new();
+
+            public override IEnumerable<FixAllScope> GetSupportedFixAllScopes()
+            {
+                yield return FixAllScope.Project;
```

Comment:
I don't think FixAll in project makes sense here - that is the default behavior for this refactoring.